frontend/src/citizen-frontend/LoginPage.tsx

```diff
@@ -73,12 +73,14 @@ export default React.memo(function LoginPage() {
               <ParagraphInfoButton
                 aria-label={i18n.common.openExpandingInfo}
                 onClick={() => setShowInfoBoxText(!showInfoBoxText)}
+                open={showInfoBoxText}
               />
             </P>
             {showInfoBoxText && (
               <ExpandingInfoBox
                 info={i18n.loginPage.applying.infoBoxText}
                 close={() => setShowInfoBoxText(false)}
+                closeLabel={i18n.common.close}
```

Comment:
How would you feel changing `ExpandingInfoBox` to default the close label to this text?

This change is everywhere, and I can't figure out a reason for not making it the default. As it is now, whenever someone adds an `ExpandingInfoBox` to a layout, they need to remember to add the default close label or we have worse accessibility.

Reply:
`ExpandingInfoBox` is in `lib-components`, and as such doesn't have access to translations.

Reply:
well that sucks 😆

Reply:
Yeah, the same applies for `ReturnButton.label`, `ExpandingInfo.ariaLabel`, etc.; a shared translations file (and i18n system) with some common translation keys between all the frontends would fix this, but I think it's out of scope for this PR.